src/view/cli.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { Header, Title } from '../component/header';
+import Text from '../component/text';
+import { Button, BackButton } from '../component/button';
+import { color, font } from '../component/style';
+
+//
+// CLI View
+//
+
+const styles = StyleSheet.create({
+  content: {
+    alignItems: 'flex-start',
+    paddingBottom: 30,
+    paddingLeft: 50,
+    paddingRight: 50,
+  },
+});
+
+const CLIView = ({ store, nav }) => (
+  <Background color={color.blackDark} style={styles.wrapper}>
+    <Header separator>
+      <BackButton onPress={() => nav.goSettings()} />
+      <Title title="CLI" />
+      <Button disabled onPress={() => {}} />
+    </Header>
+    <Background color={color.cliBackground}>
+      <MainContent style={styles.content}>
+        {store.logs.map((log, i) => <LogItem text={log} key={i} />)}
+      </MainContent>
```

Comment:
The expected behavior would be for the list to always scroll to the end like a terminal. React Native `ScrollView` and `ListView` both expose a `scrollToEnd` method: https://facebook.github.io/react-native/docs/listview#scrolltoend.

Here's how you would use that: https://stackoverflow.com/questions/42313308/react-native-listview-scrolltoend-it-doesnt-work

The `MainContent` wrapper uses a `ScrollView` under the hood which just renders all child components at once. Given that logs will probably become very long, using our `List` component will probably be faster. Might be worth testing with 100 LogItems and see how ScrollView performs.

In any case we should probably splice the `store.logs` array when items are pushed in the LogAction to limit to 100 max.